frontend/src/EmbeddedApp.tsx

```diff
@@ -1,12 +1,13 @@
-/* Use of this software is governed by the Business Source License
+/**
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
  * included in the file https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md
  *
  * As of the Change Date specified in that file, in accordance with
  * the Business Source License, use of this software will be governed
  * by the Apache License, Version 2.0
- */
-
-import { useEffect } from 'react';
+ */import { useEffect } from 'react';
```

Comment:
```suggestion
 */
 
 import { useEffect } from 'react';
```

line break between import and header